code/__HELPERS/game.dm

```diff
@@ -197,23 +189,17 @@
 	if(!T)
 		return hear
 
-	var/list/range = hear(R, T)
-
-	for(var/atom/A in range)
+	for(var/atom/A in hear(R, T))
 		if(ismob(A))
 			var/mob/M = A
 			if(M.client || include_clientless)
 				hear += M
-			//log_world("Start = [M] - [get_turf(M)] - ([M.x], [M.y], [M.z])")
-		else if(isradio(A))
```

Comment:
Any concerns about losing the radio checks? Are there any cases in which we might have been counting on this functionality?

Reply:
Nope! Nothing actually parses radios